dataverse/dataset.py

```diff
@@ -35,7 +40,8 @@ def __init__(self, entry=SWORD_BOOTSTRAP, dataverse=None, edit_uri=None,
         self._id = None
 
         # Updates sword entry from keyword arguments
-        for key, value in kwargs.iteritems():
+        for key in kwargs:
+            value = kwargs[key]
```

Comment:
Here, ``six.py`` also helps. The issue is that ``iteritems`` is called ``items`` in Python 3. The ``six.py`` file would provide an iterator called ``six.iteritems`` which does the right thing on all Python versions.